### PR TITLE
Fix CORS errors

### PR DESCRIPTION
**Pull request by krashish8**

### CODE REVIEW

File: codefest/settings.py [hunk 1]
```diff
@@ -167,16 +167,18 @@
 default_app = firebase_admin.initialize_app(cred)
 
 CORS_ORIGIN_WHITELIST = (
-    '*.codefest.tech',
-    '127.0.0.1:8080',
-    'localhost:8000',
-    '0.0.0.0:8080',
+    'https://*.codefest.tech',
+    'https://codefest.tech',
+    'http://http://127.0.0.1:8080',
```
Comment (shivanshs9) [inline codefest/settings.py:172]: lol :rofl: 
Btw, no need of HTTP or https

File: codefest/settings.py [hunk 2]
```diff
@@ -167,16 +167,18 @@
...
+    'http://localhost:8000',
+    'http://0.0.0.0:8080',
 )
 CSRF_TRUSTED_ORIGINS = (
-    '*.codefest.tech',
-    '127.0.0.1:8080',
-    'localhost:8000',
-    '0.0.0.0:8080'
+    'https://*.codefest.tech',
```
Comment (shivanshs9) [inline codefest/settings.py:177]: Check this out - https://docs.djangoproject.com/en/3.0/ref/settings/#csrf-trusted-origins

**Comment by shivanshs9** [comment]
@krashish8 you can have both CORS_ORIGIN_WHITELIST and CORS_ORIGIN_REGEX_WHITELIST as seen by the library code:

https://github.com/adamchainz/django-cors-headers/blob/31b9c2ef8a333a40f18081ffc1f1cba9fb34574d/src/corsheaders/middleware.py#L148-L153

File: codefest/settings.py (hunk 1 above)
Comment (krashish8) [reply codefest/settings.py:172]: This was done by mistake :sweat_smile:
I didn't see this :sweat_smile:

**Comment by krashish8** [comment]
@shivanshs9 Is there any other change required :sweat_smile: ? Btw, I have included HTTP or HTTPS in CORS_ORIGIN_REGEX_WHITELIST - shall I remove that?

File: codefest/settings.py [hunk 3]
```diff
@@ -166,18 +166,28 @@
 cred = credentials.Certificate(os.path.join(BASE_DIR,'service_account.json'))
 default_app = firebase_admin.initialize_app(cred)
 
+CORS_ORIGIN_REGEX_WHITELIST = (
+    r'^https://\w+\.codefest\.tech$',
+    r'^https://codefest\.tech$',
```
Comment (shivanshs9) [inline codefest/settings.py:171]: No need to repeat the domains already whitelisted in CORS_ORIGIN_WHITELIST

File: codefest/settings.py [hunk 4]
```diff
@@ -166,18 +166,28 @@
...
+    r'^http://127\.0\.0\.1:8080$',
+    r'^http://localhost:8000$',
+    r'^http://0\.0\.0\.0:8080$',
+)
 CORS_ORIGIN_WHITELIST = (
-    '*.codefest.tech',
+    '.codefest.tech',
```
Comment (shivanshs9) [inline codefest/settings.py:177]: I doubt this will work

Comment (krashish8) [reply codefest/settings.py]: Ok. Done!

**Pull request merged**